operator/internal/controller/vectorized/testdata/minimal_cluster.golden.yaml

```diff
@@ -4,15 +4,14 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  annotations:
-    redpanda.com/last-applied: <removed>
```

Comment:
This removal and the name of the use of label normalization are the only things that I'm really concerned about. Do you know if anything uses this?